### ISSUE TITLE
Failed assertion: line 779 pos 7: '_ticker != null'

### ISSUE THREAD

**Issue by IslamRustamov** [flame-engine/flame_splash_screen#11]
Installed this lib and getting this error from time to time. It's not crashing an app or something, but still might be something to investigate into. 

```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: 'package:flutter/src/animation/animation_controller.dart': Failed assertion: line 779 pos 7: '_ticker != null': AnimationController.stop() called after AnimationController.dispose()
AnimationController methods should not be used after calling dispose.
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:51:61)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)
#2      AnimationController.stop (package:flutter/src/animation/animation_controller.dart:779:7)
#3      AnimationController.value= (package:flutter/src/animation/animation_controller.dart:367:5)
#4      __SplashScreenStepState.startAnimation (package:flame_splash_screen/src/splash.dart:162:9)
<asynchronous suspension>
```

This is how my widget looks like:

```
    return MaterialApp(
      title: 'Some title',
      home: FlameSplashScreen(
        theme: FlameSplashTheme.dark,
        showAfter: (BuildContext context) {
          return const CreatedBy();
        },
        onFinish: (BuildContext context) => Navigator.pushReplacement(
          context,
          PageRouteBuilder(
            pageBuilder: (_, __, ___) => const GamePage(),
            transitionDuration: const Duration(seconds: 2),
            transitionsBuilder: (_, a, __, c) =>
                FadeTransition(opacity: a, child: c),
          ),
        ),
      ),
    );

```

**Comment by kjxbyz** [comment]
same issue.

**Comment by spydon** [comment]
This is fixed (not yet released though), and the code has been moved  here:
https://github.com/flame-engine/flame/tree/main/packages/flame_splash_screen

**Comment by spydon** [comment]
Released the fix in 0.2.0
